### PR TITLE
mi-go can now select any sound in the codebase (except music and stuff)

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
uses `SSsounds.all_sounds_except_long` instead of a manual list
also `PlayRandomSound()` uses this now too
## Why It's Good For The Game
god this list annoyed the crap out of me, also interesting to expand it to all other sounds in the game.
## Changelog
:cl: grungussuss 
sound: mi-go will now use any sound in the game
/:cl:

### CODE REVIEW

**Comment by Rhials** [comment]
The gigantic Mi-Go sound list has finally been slain at a terrible cost.

Also for linters, `find_all_sfx_except_long()` doesn't appear to have been implemented anywhere.

File: code/controllers/subsystem/sounds.dm [hunk 1]
```diff
@@ -59,7 +62,35 @@ SUBSYSTEM_DEF(sounds)
 		".aiff",
 	)
 
-	all_sounds = pathwalk("sound/", valid_file_extensions)
+	all_sounds += pathwalk("sound/ambience", valid_file_extensions)
+	all_sounds += pathwalk("sound/music", valid_file_extensions)
+	all_sounds += pathwalk("sound/announcer", valid_file_extensions)
+
+	/datum/controller/subsystem/sounds/proc/find_all_sfx_except_long()
```
Comment (Ghommie) [inline code/controllers/subsystem/sounds.dm:69]:
```suggestion
/datum/controller/subsystem/sounds/proc/find_all_sfx_except_long()
```

Comment (LemonInTheDark) [inline code/controllers/subsystem/sounds.dm:69]: how do we know these aren't long

Comment (Ghommie) [reply code/controllers/subsystem/sounds.dm:69]: This tbh, but I wouldn't be surprised if there wasn't an easy way to check it.

**Comment by LemonInTheDark** [comment]
it does?

**Pull request closed without merging**